datafusion/sql/src/parser.rs

```diff
@@ -627,6 +677,48 @@ mod tests {
             "CREATE EXTERNAL TABLE t(c1 int) STORED AS CSV PARTITIONED BY (p1 int) LOCATION 'foo.csv'";
         expect_parse_error(sql, "sql parser error: Expected ',' or ')' after partition definition, found: int");
 
+        // positive case: additional options (one entry) can be specified
+        let sql =
+            "CREATE EXTERNAL TABLE t STORED AS x OPTIONS ('k1' 'v1') LOCATION 'blahblah'";
```

Comment:
I expected to see `'k1' = 'v1'` here. Is there a particular SQL dialect being emulated here?

Reply:
That is what the code originally did. After [feedback](https://github.com/apache/arrow-datafusion/issues/4125#issuecomment-1305826215) from @avantgardnerio I changed to follow postgresql dialect (https://www.postgresql.org/docs/current/sql-createforeigntable.html)